styles.css

```diff
@@ -1,9 +1,45 @@
 body {
-  font-family: 'Roboto', sans-Serif;
-}
+  font-family: 'Roboto', sans-serif;
+  margin: 0; }
+
 .logo {
-  font-family: 'PT Serif', serif;
-}
-.main-content__title {
-  font-family: 'PT Serif', serif;
-}
+  font:bold 24px/31px pt_serifbold, arial, helvetica, sans-serif;
+  padding: 40px 0;
+  padding-left: 44px;
+  position: relative; }
+
+.logo__circle {
+  width: 28px;
+  height: 28px;
+  position: absolute;
+  left: 0;
+  top: 50%;
+  transform: translateY(-50%);
+  border-radius: 50%;
+  opacity: .7; }
+
+.logo__circle_color_blue {
+  left: 8px;
+  top: 56%;
+  background: #c8dbfb; }
+
+.logo__circle_color_dark-blue { background: #0056ec; }
+
+.main-content__title { font-family: 'PT Serif', serif; }
+
+.wrapper {
+  margin: auto;
+  width: 80%; }
+
+.header {
+  display: flex;
+  justify-content: space-between; }
+
+.menu {
+  display: flex;
+  align-items: center; }
+
+.menu__item { padding: 0 13px; }
+
+.menu__item: last-child {
```

Comment:
Codacy found an issue: [Expected newline before "}" of a multi-line block (block-closing-brace-newline-before)](https://app.codacy.com/gh/WebHeroSchool/FirstWebsait/pullRequest?prid=6170621)